masterlist.yaml

```diff
@@ -28237,9 +28237,8 @@ plugins:
         util: 'TES5Edit v4.0.1'
   - name: 'Karthwasten.esp'
```

Comment:
Okay, didn't notice that, it's for the wrong file.
I'll have to revert that first and then it can be corrected.